src/test/kotlin/racingcar/service/RacingServiceTest.kt

```diff
@@ -21,36 +20,47 @@ internal class RacingServiceTest {
         this.racingService = RacingService()
     }
 
+    @ParameterizedTest
+    @ValueSource(ints = [9, 6, 4])
+    fun `발행된 전진 확률이 4~9일 때 isMove 메서드가 true를 반환합니다`(moveProbability: Int) {
```

Comment:
```suggestion
    fun `발행된 전진 확률이 4이상일 때 isMove 메서드가 true를 반환합니다`(moveProbability: Int) {
```
"이상"과 "미만" 과 같은 단어를 활용해보면, 함수명만 보고도 로직을 이해하기가 편해져요 (이하 관련 내용 동일)

Reply:
감사합니다! 이상, 이하, 미만, 초과 등의 단어를 사용하도록 수정하였습니다!